hub/core/dataset/dataset.py

```diff
@@ -1361,9 +1362,17 @@ def pytorch(
 
         Returns:
             A torch.utils.data.DataLoader object.
+
+        Raises:
+            EmptyTensorInTheDatasetError - when one of the tensors is empty
         """
         from hub.integrations import dataset_to_pytorch as to_pytorch
 
+        # check whether we have an empty tensor inside of tensors
+        for tensor_name in self.tensors.keys():
+            if len(self.tensors[tensor_name]) == 0:
+                raise EmptyTensorInTheDatasetError
```

Comment:
```suggestion
        for tensor_name in self.tensors.keys():
            if len(self.tensors[tensor_name]) == 0:
                raise EmptyTensorInTheDatasetError(tensor_name)
```

Reply:
Reason why I chose exactly this name is that, there already exists an exception with name EmtpyTensorError

Reply:
It is used in a bit different situation. It is thrown when all tensors are empty, and shouldn't be thrown when only tensor is empty. In my case I have totally different situation. I need to throw error if only one of the tensors is empty.